processing/src/main/java/org/apache/druid/query/groupby/strategy/GroupByStrategyV1.java

```diff
@@ -233,7 +235,8 @@ public Sequence<ResultRow> apply(Interval interval)
                         outerQuery.withQuerySegmentSpec(
                             new MultipleIntervalSegmentSpec(ImmutableList.of(interval))
                         ),
-                        new IncrementalIndexStorageAdapter(innerQueryResultIndex)
+                        new IncrementalIndexStorageAdapter(innerQueryResultIndex),
+                        null
```

Comment:
I passed null here since passing the query metrics needed changes in the interface - and since this execution is synthetic over the inner query results and would always be non-vectorized (due to the storage adapter used) I skipped it. If you think, I can try to make that change